lib/workers/mixins/web_server_worker_mixin.rb

```diff
@@ -10,8 +10,6 @@ def do_work
 
   def do_before_work_loop
     @worker.release_db_connection
-
-    # Since thin traps interrupts, log that we're going away and update our worker row
```

Comment:
@jrafanie looks like it does trap SIGINT/SIGTERM: https://github.com/puma/puma/blob/6479e6b26b2b4ac7d09a78cd7b1e04470d8a213e/lib/puma/cluster.rb#L185

perhaps i'll put the comment back but s/thin/puma for this PR?